cms_form_compassion/__manifest__.py

```diff
@@ -43,14 +43,16 @@
         "queue_job",  # oca_addons/queue
         "base_automation",  # source/addons
         "link_tracker",  # source/addons
-        "base_location"  # oca_addons/partner-contact
+        "base_location",  # oca_addons/partner-contact
+        "message_center_compassion"
```

Comment:
I wouldn't make it depend just to place the settings view. Please put this settings in some existing view from the existing dependencies or base views.

Reply:
This is done by adding the setting in the base ones.